### ISSUE TITLE
Configure Security Notification

### ISSUE THREAD

**Issue by tassoman** [piwik/piwik#21168]
When action are performed inside matomo, a "SecurityNotification" is trigged by `CoreAdminHome ` plugin.

## Summary
Inside global.ini.php there is no option to disable security notification sending.  
Hooking with a plugin it's impossible to stop other plugin activiites.

My environment is still 4.14.2 stable but also appears into 5.x branch.

## Your Environment
* Matomo Version: 4.14.x + 5.x
* PHP Version: 7.4
* Server Operating System: centos7
* Additionally installed plugins: custom authentication plugin extends the core's Login plugin

**Comment by tassoman** [comment]
In my case, an user got deleted by admin interface UI then an unwanted mail got triggered. There's no trace about notifications inside CoreAdminHome plugin

I found this part of API code
https://github.com/matomo-org/matomo/blob/c94c54f62d5da04a24d84831e277449361189c78/plugins/CorePluginsAdmin/API.php#L131

**Comment by sgiehl** [comment]
You're right. It's currently not possible to turn off all or certain security emails using the config.
As a workaround you might be able to solve that by hooking into the `Mail.shouldSend` event. You could check if the mail is from a certain class and stop sending the mail in that case maybe.

**Comment by tassoman** [comment]
I found this question is also related to #18216
